roles/cloud-bootstrap/tasks/main.yml

```diff
@@ -3,3 +3,8 @@
   include: network.yml
   tags:
     - network
+
+- name: Provision users and projects
+  include: create-users-projects.yml
+  tags:
+    - create-users-projects
```

Comment:
So actually this is something we'll need to figure out once the network stuff merges... is how we create the bonnyci project here, and use it in network.yml when creating those resources. Currently sifting thru the os modules+shade+os-client-config to see if there's a way we can just specify it in the play, but it looks like we may need to keep multiple clouds.yaml entries for each project